libnetdata/libnetdata.h

```diff
@@ -312,5 +312,6 @@ extern char *netdata_configured_host_prefix;
 #include "url/url.h"
 #include "json/json.h"
 #include "health/health.h"
+#include "string/utf8.h"
```

Comment:
We also use cmake to do compilation of netdata, this change is not enough to bring a new header.
Tip: Search where a header file that is inside libnetdata is declared in the Netdata project.

Reply:
Yep clear. Thanks for pointer. Thought it will be something like that. Will fix when at pc again. Went away after the push.

Reply:
Done in #6d101b4